app-modules/prospect/src/Filament/Resources/ProspectResource/Pages/Concerns/HasProspectHeader.php

```diff
@@ -101,7 +101,8 @@ protected function getHeaderActions(): array
             ProspectTagsAction::make()->visible(fn (): bool => auth()->user()?->can('prospect.*.update')),
             ConvertToStudent::make()->visible(fn (Prospect $record) => ! $record->student()->exists()),
             DisassociateStudent::make()->visible(fn (Prospect $record) => $record->student()->exists()),
-            SubscribeHeaderAction::make(),
+            SubscribeHeaderAction::make()
+                ->view('student-data-model::filament.resources.educatable-resource.subscribe-header-action', ['record' => $this->getRecord(), 'type' => (new Prospect())->getMorphClass()]),
```

Comment:
IMO since you are passing in the record to the blade view, you might as well just go the morph class alias of the `$record` in the blade file itself to make this usage cleaner.